css/index.css

```diff
@@ -1 +1,570 @@
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/* General styling */
+
+html {
+    font-size: 62.5%;
+}
+
+.container {
+    font-size: 1.6rem;
+	max-width: 880px;
+	margin: 0 auto;
+    /*border: 1px solid red;*/
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+
+h1 {
+	font-size: 7rem;
+}
+
+h2 {
+	font-size: 3rem;
+}
+  
+h3 {
+	font-size: 1.6rem;
+}
+  
+h5 {
+	font-size: 1.8rem;
+}
+  
+p {
+	font-size: 1.6rem;
+	line-height: 1.25;
+}
+  
+footer {
+	text-align: center;
+    margin-bottom: 1.5%;
+    
+}
+  
+  
+
+  
+.button {
+	background: white;
+	color: black;
+	border: 1px solid black;
+	padding: 1% 13%;
+	border-radius: 4px;
+    
+}
+  
+a {
+	color: gray;
+	text-decoration: none;
+}
+  
+.black-line {
+	width: 100%;
+	height: 1%;
+	margin: 1.5% 0px 1.5% 0px;
+	border-top: 1px solid black;
+}
+  
+nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: baseline;
+	margin-bottom: 5%;
+	padding: 2.6% 0;  
+}
+
+/*Home Page Specific styles */
+  
+
+#home-page header {
+	
+    display: flex;
+    justify-content: flex-end;
+    border-bottom: 1px solid black;
+    padding-bottom: 9%;
+}
+
+#home-page h1 {
+    text-align: center;
+    margin-bottom: 3%;
+}
+
+a .button {
+    margin-top: 5%;
+}
+
+.button {
+	text-align: center;
+}
+
+.centered {
+	text-align: center;
+    vertical-align: middle;
+    padding-left: 12%;
+    padding-top: 8%;
+    
+}
+
+.circle-image {
+    float: right;
+}
+  
+.top-content {
+	display: flex;
+	justify-content: flex-start;
+	margin-top: 2%;
+	padding: 2% 2% 0% 0%;
+}
+
+.two-column {
+    
+    padding-right: 1%;
+    width: 49.5%;
+}
+
+.middle-img {
+	width: 100%;
+	margin-bottom: 2%;
+}
+  
+.three-column {
+    width: 33%;
+    padding-right: 1%;
+}
+
+.bottom-content {
+	display: flex;
+	justify-content: flex-start;
+    margin: 2% 0%; 
+    
+}
+ 
+.contact h3 {
+    margin-bottom: 1%;
+    margin-top: 5%;
+}
+
+address {
+	line-height: 1.15;	 
+	padding-bottom: 1%;
+}
+  
+.contact p {
+	line-height: 2;
+	padding-bottom: 2%;
+}
+
+
+
+/*Desktop*/
+@media (min-width: 818px) {
+    .tablet {
+        display: none;
+    }  
+      
+}
+
+/*Tablet*/   
+@media (max-width: 817px) {
+
+    .container {
+        width: 100%;
+    }
```

Comment:
Was there a particular reason for having your max and min widths to be 817 and 818px?